etc/FILES_MAPPING

```diff
@@ -1,3 +1,4 @@
-export DAQ_LIB=.
+ROOT=$(realpath $(dirname $BASH_SOURCE)/..)
+export DAQ_LIB=$ROOT
```

Comment:
Don't all of these need to be adjusted, not just DAQ_LIB?  Or I guess the others are always used relative to the run directory?

Reply:
we can but they don't have to be adjusted. The other 2 are always relative to the daq root.